src/gui/graphdesignerpage.h

```diff
@@ -62,12 +64,15 @@ class GraphDesignerPage : public QMainWindow
     inline QStringList graphAttrHeader() const;
     inline QStringList graphAttrValues() const;
     inline PluginKey selectedGraphKey() const;
+    inline FullInspector* fullInspector();
```

Comment:
getter functions should be `const`, i.e.,
`inline FullInspector* fullInspector() const;`